### PR TITLE
0.1.2 beta programme announcement

### PR DESCRIPTION
**Pull request by TrechNex**

### CODE REVIEW

**Comment by TrechNex** [comment]
Yes, once again this'll be squashed 🤦‍♂

File: content/about.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ Glimpse is an open source image editor based on the GNU Image Manipulation Progr
 - [Will you support BSD, Haiku, Solaris or other UNIX platforms?](#will-you-support-bsd-haiku-solaris-or-other-unix-platforms)
 - [Will you contribute changes back upstream?](#will-you-contribute-changes-back-upstream)
 - [This project is big! Will it take a long time to rebrand?](#this-project-is-big-will-it-take-a-long-time-to-rebrand)
+- [How does you release numbering work?](#how-does-your-release-numbering-work)
```
Comment (BrainBlasted) [inline content/about.md]: *your

File: content/about.md [hunk 2]
```diff
@@ -166,6 +167,19 @@ We have been pleased by the positive feedback we have received from our userbase
 
 [Return to top](#contents)
 
+## How does you release numbering work? {#how-does-your-release-numbering-work}
```
Comment (BrainBlasted) [inline content/about.md]: *your here too

Comment (TrechNex) [reply content/about.md]: Fixed in b6d3b6f - good spot!

**Pull request merged**